sponsorship.md

```diff
@@ -12,6 +12,18 @@ SoCraTes UK is a non-profit conference organised by and for the Software Crafter
 our attendees only need to pay for their hotel room. As such, we rely heavily on our sponsors to reduce 
 the overall cost of attendance and in exchange, we provide exposure (and well deserved Kudos!) to all of our sponsors.
 
+<br>
+
+### Why Sponsor SoCraTes UK?
+
+SoCraTes brings together a community of software professionals who care about creating high-quality, maintainable software, and want to work together to share and build on their skills.  
+As an unconference with no predetermined agenda, SoCraTes is created by its participants. As a result, it attracts highly engaged, collaborative practitioners, and encourages dense networks of ideas.  
+By sponsoring SoCraTes, you can demonstrate that you share these values of collaboration and continuous improvement in software development. If you are recruiting, then this is a great way to reach out to a community of highly engaged potential candidates.  
+
+<br>
```

Comment:
Not sure those new lines are necessary?

Reply:
See https://github.com/lscc/socrates-uk/pull/74#discussion_r265917805